File: Kyu 7 Exes and Ohs/solution.py
def xo(s):
    return (lambda s: s.count('x') == s.count('o'))(s := s.lower())


if __name__ == '__main__':
    test_cases = ['xo', 'xxOo', 'xxxm', 'Oo', 'ooom', 'xxxoo']
    for test in test_cases:
        print(xo(test))

File: Kyu 7 Exes and Ohs/solution.nim
import std/strutils

var 
    testCases: array[6, string] = ["xo", "xxOo", "xxxm", "Oo", "ooom", "xxxoo"]

proc XO(testCase: string): bool = 
    var x = 0
    var o = 0
    for i in toLowerAscii(testCase):
        if i == 'x':
            x = x + 1
        elif i == 'o':
            o = o + 1
    return x == o


for testCases in testCases:
    echo XO(testCases)

    
    